sections/clause_0_front_material.adoc

```diff
@@ -1,10 +1,27 @@
 [abstract]
 == Abstract
 
-This best practice document on UML to JSON encoding rules is established as an initiative of Geonovum. The aim is to come to a standardized encoding from UML to JSON, in order to achieve technical interoperability in the chain from conceptual models to JSON implementation. In this document, JSON implementation includes plain JSON, GeoJSON and JSON-FG.
+This best practice document on UML to JSON encoding rules is initiated as an initiative of Geonovum. The aim is to come to a standardized encoding from UML to JSON, in order to achieve technical interoperability in the chain from conceptual models to JSON implementation. In this document, JSON implementation includes plain JSON, GeoJSON and JSON-FG.
```

Comment:
Suggest to rephrase to "[...] encoding rules is an initiative of Geonovum."

Reply:
agree

Reply:
done